Characters in the Http.java and Jsonp.java Javadocs cause publish-local to fail on Windows

[info] Loading source file E:/Projects/playframework/framework/src/play/src/main
/java/play/mvc/Http.java...
[error] E:\Projects\playframework\framework\src\play\src\main\java\play\mvc\Http
.java:572: error: unmappable character for encoding Cp1252
[error]          \* Set a new transient cookie with path ΓÇ£/∩┐╜?<br>
[error]                                                       ^
[error] E:\Projects\playframework\framework\src\play\src\main\java\play\mvc\Http
.java:585: error: unmappable character for encoding Cp1252
[error]          \* Set a new cookie with path ΓÇ£/∩┐╜?
[error]                                             ^

--- a/framework/src/play-java/src/main/java/play/libs/Jsonp.java
+++ b/framework/src/play-java/src/main/java/play/libs/Jsonp.java
@@ -25,7 +25,7 @@ import play.libs.Json;
  * <pre>
  *   GET  /my-service?callback=foo
  * </pre>
- * The response will have content type “text/javascript” and will look like the following:
+ * The response will have content type "text/javascript" and will look like the following:
  * <pre>
  *   foo({...});
  * </pre>

--- a/framework/src/play/src/main/java/play/mvc/Http.java
+++ b/framework/src/play/src/main/java/play/mvc/Http.java
@@ -569,7 +569,7 @@ public class Http {
         }
 
         /**
-         * Set a new transient cookie with path “/”<br>
+         * Set a new transient cookie with path "/"<br>
          * For example:
          * <pre>
          * response().setCookie("theme", "blue");
@@ -582,7 +582,7 @@ public class Http {
         }
 
         /**
-         * Set a new cookie with path “/”
+         * Set a new cookie with path "/"
          * @param name Cookie name.  Must not be null.
          * @param value Cookie value.
          * @param maxAge Cookie duration (null for a transient cookie and 0 or less for a cookie that expires now).

